### PR TITLE
Improves readme info about environment setup

### PR DESCRIPTION
**Pull request by josemigallas**
This PR solves the ticket: https://issues.jboss.org/browse/AGPUSH-2066

### CODE REVIEW

File: admin-ui/README.md [hunk 1]
```diff
@@ -2,38 +2,55 @@
...
+```
+That's because you did not deploy the correct file. After building the whole project with `mvn`, you will have this two files located in `aerogear-unifiedpush-server/servers/ups-wildfly/target/`:
+```
+ag-push.war
+ag-push
+```
+`ag-push.war` can be deployed directly but in order for Grunt to work properly, `ag-push` must be deployed instead. To do that, simply rename it to `ag-push.war` and move it to the `standalone/deployments` directory:
+```
+cd $SERVER_HOME/standalone/deployments
+cp path/to/aerogear-unifiedpush-server/servers/ups-wildfly/target/ag-push ag-push.war
```
Comment (lholmquist) [inline admin-ui/README.md]: I think that needs to be "cp -r" since you are copying a directory

Comment (josemigallas) [inline admin-ui/README.md]: You are completely right :+1:

**Comment by lholmquist** [comment]
@josemigallas i think this is looking pretty good. 👍

**Comment by matzew** [comment]
:tada:

**Pull request merged**